uint32 size
float64 min_distance
float64 cost
geometry_msgs/Point travel_point
geometry_msgs/Point initial
geometry_msgs/Point centroid
geometry_msgs/Point middle
geometry_msgs/Point[] points
